src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,136 @@ Master Pre-Releases
+  `PR #8456 <https://github.com/pantsbuild/pants/pull/8456>`_
+
+* remove @_algebraic_data decorator to support @dataclass and mypy (#8437)
+  `PR #8437 <https://github.com/pantsbuild/pants/pull/8437>`_
+
+* Fix double check cache digest (#8416)
+  `PR #8416 <https://github.com/pantsbuild/pants/pull/8416>`_
+
+* Don't panic in create_scheduler (#8428)
+  `PR #8428 <https://github.com/pantsbuild/pants/pull/8428>`_
+
+* [junit-run] apply platform args to junit run java exec (#8421)
+  `PR #8421 <https://github.com/pantsbuild/pants/pull/8421>`_
+
+* Don't shade the nailgun package (#8418)
+  `PR #8418 <https://github.com/pantsbuild/pants/pull/8418>`_
+
+* Make subsystem dep explicit for RSC compile (#8406)
+  `PR #8406 <https://github.com/pantsbuild/pants/pull/8406>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Improve support for `Enum`s as option types (#8458)
+  `PR #8458 <https://github.com/pantsbuild/pants/pull/8458>`_
+  `Comment #8451 <https://github.com/pantsbuild/pants/pull/8451#discussion_r334075761>`
+
+* Port `test_pytest_run_integration.py` to V2 (#8466)
+  `PR #8466 <https://github.com/pantsbuild/pants/pull/8466>`_
+
+* Fix `PantsRunIntegrationTest.mock_buildroot()` to work with `--chroot` (#8465)
+  `PR #8465 <https://github.com/pantsbuild/pants/pull/8465>`_
+
+* Use f-strings in python tasks. (#8449)
+  `PR #8449 <https://github.com/pantsbuild/pants/pull/8449>`_
+
+* Run fuse tests on OSX again (#8453)
+  `PR #8453 <https://github.com/pantsbuild/pants/pull/8453>`_
+  `Issue #10017 <https://github.com/travis-ci/travis-ci/issues/10017>`_
+
+* Replace majority of remaining `datatype()`s with `@dataclass` (#8435)
+  `PR #8435 <https://github.com/pantsbuild/pants/pull/8435>`_
+  `PR #8423 <https://github.com/pantsbuild/pants/pull/8423>`_
+
+* Use f-strings in pants daemon (#8448)
+  `PR #8448 <https://github.com/pantsbuild/pants/pull/8448>`_
+
+* Add collections.Enum class that extends stdlib Enum with ~pattern matching (#8451)
+  `PR #8451 <https://github.com/pantsbuild/pants/pull/8451>`_
+  `PR #8443 <https://github.com/pantsbuild/pants/pull/8443>`_
+
+* use travis wait enhanced in shards with remoting enabled (#8446)
+  `PR #8446 <https://github.com/pantsbuild/pants/pull/8446>`_
+
+* Add file to allow a clean looking git blame (#8404)
+  `PR #8404 <https://github.com/pantsbuild/pants/pull/8404>`_
+
+* Run testprojects integration tests with V2 test runner (#8426)
+  `PR #8426 <https://github.com/pantsbuild/pants/pull/8426>`_
+  `PR #8439 <https://github.com/pantsbuild/pants/pull/8439>`_
+
+* Restore running `test_testprojects_integration.py` with chroot (#8433)
+  `PR #8433 <https://github.com/pantsbuild/pants/pull/8433>`_
+  `PR #8429 <https://github.com/pantsbuild/pants/pull/8429>`_
+
+* Replace more `datatype()` instances with `@dataclass` (#8423)
+  `PR #8423 <https://github.com/pantsbuild/pants/pull/8423>`_
+  `PR #8362, <https://github.com/pantsbuild/pants/pull/8362,>`_
```

Comment:
This link is broken with a comma.

Reply:
Thanks for catching the bad links ..will clean up